srcpkgs/cpptest/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'cpptest'
+pkgname=cpptest
+version=1.1.2
+revision=1
+wrksrc=${pkgname}-${version}
```

Comment:
this is the default value of wrksrc, so you can remove this line

Reply:
Fixed.